cve_bin_tool/backports/__init__.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict
+
+import distro
+
+from cve_bin_tool.cve_scanner import CVEData
+from cve_bin_tool.log import LOGGER
+from cve_bin_tool.output_engine.util import ProductInfo
+
+from .debian_backports import DebianCVETracker
+
+DEBIAN_DISTROS = ["debian", "ubuntu"]
+
+
+class BackportReport:
+    def __init__(
+        self,
+        all_cve_data: Dict[ProductInfo, CVEData],
+    ):
+        self.all_cve_data = all_cve_data
+
+    def check_backport(self):
+        if distro.id() in DEBIAN_DISTROS:
```

Comment:
It may be useful for users to be able to list backport information even if the machine being used for scanning isn't running debian itself.  I'm thinking particularly of folk who might be running cve-bin-tool in a separate CI system or running it on a downloads server, but it might also be important for other debian-based distros whose names we don't know yet. 

Let's figure out if there's a nice way to do that.  Could backports/debian_backports.py take an argument (say from a `--backported-fixes=debian` option, and have the default be "local_system" that grabs distro.id() or something) instead of distro.id()?

Reply:
If the user specify an option `debian` how will the system recognise whether which version of debian the user wants? 🤔

Reply:
I like having `debian-bullseye` etc. as options.  Not sure the best way to communicate the availability of these options to users, but maybe just documenting it in Manual.md?

Reply:
The help message got very messy from the long list of options. So I'll add these options in MANUAL.md.